clang/test/Headers/float.c

```diff
@@ -224,7 +224,8 @@
     #error "Mandatory macro NAN is missing."
   #endif
   // FIXME: the NAN diagnostic should only be issued once, not twice.
```

Comment:
This comment should be updated because now we're issuing the inf *and* nan warnings twice instead of once.